### PR TITLE
Update dependencies

### PR DESCRIPTION
**Pull request by dasanra**
Bump and optimize dependencies to avoid version mismatch between `safe-react` and `safe-react-components`

Remove unused dependencies
Move `*-loader` dependencies to `devDependencies`

### CODE REVIEW

**Comment by gnosis-info** [comment]
Travis automatic deployment:
 https://pr75--safereactcomponents.review.gnosisdev.com

**Comment by gnosis-info** [comment]
Travis automatic deployment:
 https://pr75--safereactcomponents.review.gnosisdev.com

File: .storybook/main.js [hunk 1]
```diff
@@ -1,7 +1,6 @@
 module.exports = {
   stories: ["../src/**/*.stories.tsx"],
   addons: [
-    "@storybook/preset-create-react-app",
```
Comment (nicosampler) [inline .storybook/main.js:4]: :eyes:

**Review by nicosampler — APPROVED** [review]

**Comment by gnosis-info** [comment]
Travis automatic deployment:
 https://pr75--safereactcomponents.review.gnosisdev.com

**Comment by gnosis-info** [comment]
Travis automatic deployment:
 https://pr75--safereactcomponents.review.gnosisdev.com

**Comment by gnosis-info** [comment]
Travis automatic deployment:
 https://pr75--safereactcomponents.review.gnosisdev.com

**Pull request merged**